documentation/01-Prerequisites.md

```diff
@@ -151,17 +144,17 @@ sudo apt-get install exuberant-ctags
 
 > When available `make ctags` can be invoked to help generate a `tags` database at the root of the source tree to allow easier code navigation in editors that support it.
 
-### Linux Python Install
+#### Linux Python Install
```

Comment:
```suggestion
### Linux Python Install
```

Python can be it's own thing in a few of the light weight examples.  They also aren't really build tools (e.g. not required for the C++/C# side of things).
Also, if we next Python install under Manual Build Tool Install, then the Docker part doesn't make as much sense.

Maybe instead, options 1/2 (docker vs manual) should be nested under a more generic "Linux Build Tools" header, and then after that header a "Linux Python Install" header which has the docker vs. conda vs. manual subsections.